### ISSUE TITLE
Invalid location of Base tag

### ISSUE THREAD

**Issue by serega270** [quasarframework/quasar#4846]
I config the `quasar.conf.js,` as below:
```
build: {
      scopeHoisting: true,
      vueRouterMode: 'history',
      publicPath: '/ ,
      gzip: true,
      ...
}
```

it seems that the **base** tag should be declared before other links

```
<!DOCTYPE html>
<html>
<head><title>Quasar App</title>
  <meta charset=utf-8>
  <meta name=description content="A Quasar Framework app">
  <meta name=format-detection content="telephone=no">
  <meta name=msapplication-tap-highlight content=no>
  <meta name=viewport content="user-scalable=no,initial-scale=1,maximum-scale=1,minimum-scale=1,width=device-width">

<!-- Correct place for <base href="/" > -->

  <link rel=icon type=image/png href=statics/app-logo-128x128.png>
  <link rel=icon type=image/png sizes=16x16 href=statics/icons/favicon-16x16.png>
  <link rel=icon type=image/png sizes=32x32 href=statics/icons/favicon-32x32.png>
  <link rel=icon type=image/png sizes=96x96 href=statics/icons/favicon-96x96.png>
  <link rel=icon type=image/ico href=statics/icons/favicon.ico>
  <link href=/js/2d22c0ff.868145e9.js rel=prefetch>
  <link href=/js/461a5054.17d4a942.js rel=prefetch>
  <link href=/js/4b47640d.65105906.js rel=prefetch>
  <link href=/css/app.de49d83c.css rel=preload as=style>
  <link href=/js/app.995a58e1.js rel=preload as=script>
  <link href=/js/runtime.514e2f10.js rel=preload as=script>
  <link href=/js/vendor.ad270399.js rel=preload as=script>
  <link href=/css/app.de49d83c.css rel=stylesheet>
  <base href=/>
</head>
<body>
<div id=q-app></div>
<script type=text/javascript src=/js/app.995a58e1.js></script>
<script type=text/javascript src=/js/runtime.514e2f10.js></script>
<script type=text/javascript src=/js/vendor.ad270399.js></script>
</body>
</html>
```
@quasar/app v1.0.4

**Comment by rstoenescu** [comment]
Hi,

Any reference to this? From what I know it doesn't matter.

**Comment by hawkeye64** [comment]
@serega270 You can reference this: https://html.spec.whatwg.org/dev/urls-and-fetching.html#base-urls
Order does not matter

**Comment by serega270** [comment]
@rstoenescu 
https://jsfiddle.net/qekdx1w5/1/
![image](https://user-images.githubusercontent.com/15049964/62729838-ce5de500-ba2f-11e9-9333-00d259550690.png)

========
The same without sandbox
![image](https://user-images.githubusercontent.com/15049964/62730557-424cbd00-ba31-11e9-9c0a-be67672c8c47.png)

**Comment by rstoenescu** [comment]
jsFiddle does not work as you'd imagine. the base tag does NOT end up in head.

<img width="387" alt="Screenshot at Aug 08 22-43-46" src="https://user-images.githubusercontent.com/4665641/62732893-167c0800-ba2e-11e9-818a-7cd8c06153d3.png">

**Comment by serega270** [comment]
@rstoenescu yes, sorry, jsFiddle is bad example
but please look the example without sandbox
![image](https://user-images.githubusercontent.com/15049964/62733311-5f848a00-ba37-11e9-9d34-974b6aef8857.png)

![image](https://user-images.githubusercontent.com/15049964/62733327-68755b80-ba37-11e9-8257-ac06f09b5ee0.png)

**Comment by rstoenescu** [comment]
Will be fixed in quasar/app 1.0.6.
